Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This is dev version to make the Cargo builds fast and should re-build and run the whole
+# app everytime the container is restarted, as Rust doesn't has a proper way of hot reloading
+FROM rust:latest
```

Comment:
Yeah @swarnimarun, you are right here. But my intention was to use the latest Rust due to Stable changes that Rust adds up in every release. Also, Since Rust is evolving and has many up-to-date upgrade in latest release, I thought it is better to use `latest` image of Rust.